### PR TITLE
fix: extended README.md

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of https://github.com/artipie/artipie/issues/1031 

Corrected and extended readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,21 +46,49 @@ This is the dependency you need:
 </dependency>
 ```
 
-Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto)
-for more technical details.
+Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto) for more technical details. If you 
+have any question or suggestions, do not hesitate to create as issue, our roadmap is
```
Comment (dgarus) [inline README.md:50]: Maybe we should add the link to artipie's telegram chat?

File: README.md [hunk 2]
```diff
@@ -45,21 +46,49 @@ This is the dependency you need:
...
  - `Content` represents storage binary value
 
-[Storage](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Storage.html)
-[Key](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Key.html) and other types are
+[Storage](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Storage.html),
+[Key](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Key.html) and other entities are
 documented in [javadoc](https://www.javadoc.io/doc/com.artipie/asto/latest/index.html).
 
-TO DO: add more details
+Here is en example of how to create `FileStorage`, save and then read some data:
```
Comment (dgarus) [inline README.md:64]: `FileStorage` example is well, but there is no description how to use s3 storage.
Maybe we should create a wiki for asto?

**Comment by olenagerasimova** [comment]
@dgarus thanks, added example for `S3Storage` and link to telegram chat

File: README.md [hunk 3]
```diff
@@ -45,21 +46,63 @@ This is the dependency you need:
 </dependency>
 ```
 
-Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto)
-for more technical details.
+Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto) for more technical details. If you 
+have any question or suggestions, do not hesitate to create as issue or contact us in 
+[Telegram](https://t.me/artipie), our roadmap is available [here](https://github.com/orgs/artipie/projects/3/views/1).
```
Comment (dgarus) [inline README.md:51]: Think, `our roadmap` can be a separate sentence.
Shorter link `https://github.com/orgs/artipie/projects/3`.

File: README.md [hunk 4]
```diff
@@ -45,21 +46,63 @@ This is the dependency you need:
...
+    val -> new PublisherAs(val).asciiString()
+).thenAccept(
+    System.out::println
+).join();
+```
+In the example we created local text file `/usr/local/example/hello.txt` containing string "Hello world!",
+then read and print it into console. Used classes:
+- `Key.From` is implementation of the `Key` interface, keys are strings, separated by `/`
+- `Content.From` implements `Content` interface, allows to create `Content` instances from 
+  byte arrays or publisher
```
Comment (dgarus) [inline README.md:83]: publisher -> publisher of ByteBuffer's.
And link for publisher: https://www.reactive-streams.org/reactive-streams-1.0.4-javadoc/org/reactivestreams/Publisher.html.

What do you think?

**Comment by olenagerasimova** [comment]
@dgarus thanks, corrected

**Pull request merged**